build.sbt

```diff
@@ -6,10 +6,10 @@ name := "facia-api-client"
 
 description := "Scala client for The Guardian's Facia JSON API"
 
-ThisBuild / scalaVersion := "2.13.14"
+ThisBuild / scalaVersion := "2.13.15"
 
 val sonatypeReleaseSettings = Seq(
-  releaseVersion := fromAggregatedAssessedCompatibilityWithLatestRelease().value,
+  // releaseVersion := fromAggregatedAssessedCompatibilityWithLatestRelease().value,
```

Comment:
We have to comment this out due to this issue, as we're introducing the new `fapi-client-core` artifact:

* https://github.com/guardian/gha-scala-library-release-workflow/issues/33

...we'll re-enable it immediately after the next release.

Reply:
Re-enabled with 42fa0bd1e44a59c4270f78fa305de7063eb0c044.

Reply:
Sadly this created the impression that this release was much more compatible with the previous release than it actually is, making https://github.com/guardian/mobile-apps-api/pull/3503 a bigger piece of work than I was expecting.

Reply:
You're right, this PR (https://github.com/guardian/facia-scala-client/pull/287) was released as a patch release ([v14.0.1](https://github.com/guardian/facia-scala-client/releases/tag/v14.0.1)) when it should have gone out as a major release (because it _isn't_ binary or source compatible), bumping the first integer of the version number.

> making [guardian/mobile-apps-api#3503](https://github.com/guardian/mobile-apps-api/pull/3503) a bigger piece of work than I was expecting.

I've updated the release notes for v14.0.1 to list the significant incompatibilities found in this PR.